CONTRIBUTING.md

```diff
@@ -54,7 +53,7 @@ instead.
 
 ## Contributing to Kong documentation and the Kong Hub
 
-Improving the documentation is a most welcome form of contribution to the Kong
+Improving the documentation is the most welcome form of a contribution to the Kong
```

Comment:
The sentence is grammatically correct and the usage is fine, please revert this change.

"A most welcome" is a common phrase. https://sentence.yourdictionary.com/welcome

"Form of [noun]" refers to a mode of an abstract concept, as opposed to "form of a [noun]," a specific event or entity.